"""
Custom Error for SQLAlchemy failing to query the database.
"""

class QueryExecutionError(Exception):
    pass